_extras/Recipe.md

```diff
@@ -0,0 +1,75 @@
+### Documentation
+* **Create an Etherpad** or other collaborative document, and add all vital information to be included in your workshop introduction for learners who arrive late. This should include:
+    * Workshop schedule (including the times of all breaks)
+    * Connection instructions
+    * What to do in the event of a connection failure
+    * How learners can ask questions (e.g. Zoom chat, raise hand, etc.)
+    * Other instructions related to workshop mechanics, including recommended window arrangement
+    * Links to The Carpentries Code of Conduct as well as your workshop website
+* Add a link to your Etherpad to your **workshop website**.
+* For a self-organised workshop, be sure to submit your [workshop request form](https://amy.carpentries.org/forms/self-organised/).
+* **Email your learners** with everything they need to know in advance of the workshop. 
+    * Our [template for in-person trainings](https://docs.carpentries.org/topic_folders/workshop_administration/email_templates.html#email-learners-before-workshop) may be useful. Much of this info may already be on your Etherpad, so you can be selective about information that needs duplicating.
+### Preparing for Adjustments
+* **Plan to cut content.** Online workshops often run more slowly than they do in-person; sometimes much more slowly. Decisions to cut back should be intentional and collaborative.
+    * Remember that skipping steps in a workflow can create problems with dependencies later on.
+* **Agree on your priorities.** Discussing key objectives and broader goals with your instructional team in advance will help everyone navigate difficult trade-offs when time becomes an issue during a workshop.
+* **Have a backup** for all roles and documents. Technical failures come in all shapes and sizes, and life can still happen, too!
+* **Review your pre-workshop surveys** using the links provided in your email from our workshop administrators. Knowing your audience will help you adjust appropriately to their needs!
+### Update your Software
+* Especially for Zoom, you will want to be using the most recent version and the same version as everyone else. Remind your learners to update as well!
+## 2. Workshop Mechanics
+* **Assign a "Zoom DJ"** to assume the "host" role in the meeting. A Zoom DJ will manage breakouts, waiting rooms, muting, and troubleshooting as needed. This role is typically played by a Supporting Instructor; if/when they choose to teach, plan to hand off this role to another Instructor.
+* **Customize profile names** to clearly indicate instructors and helpers. For example, a helper named Ada might change their profile to read "Helper: Ada (she/her)"
+* **Use Zoom features as you would use sticky notes**, and remember to call upon learners to use them regularly. 
+    * On Zoom, learners can choose "reactions" that include a green '✓' and a red 'x' icon. Unlike other reactions, these persist until cleared.
```

Comment:
Just to add that most of this document applies to Microsoft Teams, especially after they introduced breakout rooms. It's a bit slower to open/close them then in Zoom, but they work. What I miss in teams is that it has no persistent reactions. A way to simulate them is to paste different messages (e.g. with 'Yes' and 'No' or with answer numbers) in the meeting chat, and ask learners to like the message corresponding to their answer.